packages/devtools_app/lib/src/screens/provider/instance_viewer/instance_providers.dart

```diff
@@ -195,26 +197,24 @@ Future<EnumInstance?> _tryParseEnum(
 
   final _nameRef = findPropertyWithName('_name');
   final indexRef = findPropertyWithName('index');
-
   if (_nameRef == null || indexRef == null) return null;
 
   final nameInstanceFuture = eval.safeGetInstance(_nameRef, isAlive);
   final indexInstanceFuture = eval.safeGetInstance(indexRef, isAlive);
 
   final index = await indexInstanceFuture;
-
   if (index.kind != InstanceKind.kInt) return null;
 
   final name = await nameInstanceFuture;
   if (name.kind != InstanceKind.kString) return null;
 
+  // Some Dart versions have for name "EnumType.valueName", others only have "valueName".
+  // So we have to strip the type manually
   final nameSplit = name.valueAsString!.split('.');
 
-  if (nameSplit.length != 2) return null;
```

Comment:
This check seems to be needed. It was added because of bug: https://github.com/flutter/devtools/pull/3672

With your update the code will not fail, but the result will be wrong.

Reply:
Is it really? I also changed `nameSplit[1]` into `nameSplit.last`, so the number of split shouldn't matter now.